pontoon/messaging/utils.py

```diff
@@ -0,0 +1,10 @@
+import html2text
+
+
+def html_to_plain_text_with_links(html):
```

Comment:
I would just inline this and consider DRY much later, but that's mostly just me.

Reply:
I'm planning to use this function in the [email script](https://github.com/mozilla-l10n/pontoon-scripts/blob/c06950b4761f4208eacf39496b4388980a1f3dda/admin/email_script_events.py), at least for the time being.